package.json

```diff
@@ -1,14 +1,24 @@
 {
   "private": true,
   "scripts": {
-    "test": "lerna run test"
+    "test": "yarn prettier --list-different && yarn lint && lerna run test",
+    "lint": "eslint \"**/*.js\"",
+    "format": "yarn prettier --write",
+    "prettier": "prettier \"**/*.{js,ts,tsx}\" --loglevel warn"
```

Comment:
Double quotes are required on Windows. With single quotes both prettier and eslint couldn't fine any files.